cognite/neat/_rules/analysis/_base.py

```diff
@@ -109,14 +109,14 @@ def inherited_referred_classes(self) -> set[ClassEntity]:
         raise NotImplementedError
 
     # Todo Lru cache this method.
-    def class_parent_pairs(self) -> dict[T_ClassEntity, list[T_ClassEntity]]:
+    def class_parent_pairs(self, allow_different_namespace: bool = False) -> dict[T_ClassEntity, list[T_ClassEntity]]:
```

Comment:
Since we are checking on prefix == space
```suggestion
    def class_parent_pairs(self, allow_different_space: bool = False) -> dict[T_ClassEntity, list[T_ClassEntity]]:
```